### PR TITLE
Update doc for unions

### PR DESCRIPTION
**Pull request by tmfink**
Addresses #832

I also modified the `--help` text to print the default Rust target.

### CODE REVIEW

File: src/options.rs [hunk 1]
```diff
@@ -14,8 +14,10 @@ where
     I: Iterator<Item = String>,
 {
     let rust_target_help = format!(
-        "Version of the Rust compiler to target. Valid options are: {:?}.",
-        RUST_TARGET_STRINGS);
+        "Version of the Rust compiler to target. Valid options are: {:?}. Defaults to {:?}.",
+        RUST_TARGET_STRINGS,
+        String::from(RustTarget::default()),
+        );
```
Comment (emilio) [inline src/options.rs]: nit: Indentation of the paren should go to the left.

**Comment by fitzgen** [comment]
Thanks for the docs update!

**Comment by tmfink** [comment]
:smile:

**Pull request merged**